crates/libtiny_client/src/state.rs

```diff
@@ -719,16 +719,14 @@ async fn retry_channel_join(
 
     let mut rcv_abort = ReceiverStream::new(rcv_abort).fuse();
 
-    match timeout(Duration::from_secs(10), rcv_abort.next()).await {
-        Err(_) => {
-            // Send join message
-            snd_irc_msg
-                .try_send(wire::join(std::iter::once(channel.as_ref())))
-                .unwrap();
-        }
-        Ok(_) => {
-            // Channel tab was closed
```

Comment:
Could you bring back this comment please? Maybe use put it before the `if`, something like

```
// Send join message after timeout. Error means channel tab was closed.
if timeout(...) { ... }
```

Reply:
Done. I think `Ok` means it was closed, right?

Reply:
Right, thanks!